src/cf-expandables/src/cf-expandables.less

```diff
@@ -145,66 +140,63 @@
     //
 
     &__padded {
-        margin-bottom: unit( 30px / @base-font-size-px, em );
-        background: @expandable__padded-bg;
-
-        &:hover,
-        &.o-expandable__expanded {
-            background: @expandable__padded-bg-hover;
-        }
+        //margin-bottom: unit( 30px / @base-font-size-px, em );
 
         .o-expandable_header {
             padding: unit( 10px / @base-font-size-px, em )
                      unit( 15px / @base-font-size-px, em );
         }
 
         .o-expandable_content {
-            // Margin collapsing trick for more consistent bottom padding.
-            margin: 0
-                    unit( 15px / @base-font-size-px, em )
-                    unit( 22px / @base-font-size-px, em );
+            padding: 0 unit( 15px / @base-font-size-px, em );
 
             // The divider between _header and _content.
             &:before {
                 content: '';
                 display: block;
-                border-bottom: 1px solid @expandable__padded-divider;
-                margin-bottom: unit( 15px / @base-font-size-px, em );
+                border-top: 1px solid @expandable__border;
+                padding-top: unit( 15px / @base-font-size-px, em );
             }
 
             &:after {
-                margin-bottom: unit( 15px / @base-font-size-px, em );
+                padding-bottom: unit( 15px / @base-font-size-px, em );
                 width: 100%;
             }
         }
     }
 
+    //
+    // Expandable with a background color modifier
+    //
+    &__background {
+        background: @expandable__background;
+    }
+
+    //
+    // Expandable with a border modifier
+    //
+    &__border {
+        border: 1px solid @expandable__border;
+    }
+
     //
     // Expandable groups
     //
 
     &-group {
-        margin-bottom: unit( 30px / @base-font-size-px, em );
+        //margin-bottom: unit( 30px / @base-font-size-px, em );
 
-        .o-expandable {
-            border-bottom: 1px solid @expandable-group-divider;
-            margin-bottom: 0;
-            background: @expandable-group-bg;
+        .o-expandable__padded {
+            border-bottom: 1px solid @expandable__border;
+            //margin-bottom: 0;
 
             &:first-child {
-                border-top: 1px solid @expandable-group-divider;
+                border-top: 1px solid @expandable__border;
             }
         }
 
         .o-expandable_label {
-            .heading-4();
```

Comment:
Why remove this and use the `h4` class instead?

Reply:
Because this mixin doesn't account for the change in font size on smaller screens. See issue #779

Reply:
ok, I think it complicates the module because it gives the impression that the heading size is flexible.

Reply:
Weirdly, it is. Solo expandables don't have a modified font size or weight, it appears that's only for expandable groups. I'm not sure of the reason, but maybe @nataliafitzgerald can fill us in.

Reply:
I also should have mentioned that because we're not modifying the h4 mixin in any way that using the class in the recommended markup pattern reduces the duplication of output which is a big culprit to our css file size woes.

Reply:
yeah, it is part of my confusion surrounding when to use margin utilities within atomic components. If an interior DOM element needs some margin ( same across all breakpoints ), should we use the utility margin classes or just handle it with CSS?

Reply:
I feel like utility classes are a bit different, they do one thing, and one thing well, but that thing could also usually be packaged with the other things happening in that element to help describe the situation. IE:

```less
.m-thing {
  padding: 15px;
  background-color: @gray-5;

  a-atom {
    margin-bottom: 30px;
  }
}
```
In this example the margin is the only style that could be offloaded to the markup, providing minimal bloat reduction. On the other hand, keeping it in the Less provides the next developer a lot more context of how the pattern works as a whole.

For something complex like h# classes that include font styles, margin, letter-spacing, and breakpoint changes, putting them into the markup removes far more bloat and the font-styles don't have any affect on the rest of the pattern so there isn't a loss of context by removing them from the Less.